### PR TITLE
update medley README to match new build instructions

### PR DESCRIPTION
**Pull request by masinter**

### CODE REVIEW

**Comment by MattHeffron** [comment]
There is a _**NUL**_ character on line 13 _in_ the word **be**.
Line 68 unmatched right parenthesis: **difficulty.)**
Line 115: **at least in the library, lispusers.**  => **at least in the library and lispusers directories.**

Are the Docker related bits relevant? (Are they used for the online environment?)

**Comment by nbriggs** [comment]
Removed the NUL so now you can compare the changes on github.

File: README.md [hunk 1]
```diff
@@ -59,15 +57,17 @@ Unpack the medley loadups file
...
 
 Mac users should get [XQuartz from XQuartz.org](https://xquartz.org/releases).
 
 Medley manages the display entirely, doesn't use X fonts and manages it's own window system.
 
-If you have a high-resolution display, note that much of the graphics was designed for a low-resolution display, so an X-server that does "pixel doublilng" is best. (E.g., Raspberry Pi does pixel doubling on 4K displays.) It also presumes you have a 3-button mouse; the scroll-wheel on some mice act as one with some difficulty.) XQuartz Preferences/Input has "Emulate three button mouse" option.
+If you have a high-resolution display, note that much of the graphics was designed for a low-resolution display, so an X-server that does "pixel doublilng" is best. (E.g., Raspberry Pi does pixel doubling on 4K displays.)
+
+Medley presumes you have a 3-button mouse; the scroll-wheel on some mice act as one with some difficulty.) Go  into XQuartz Preferences/Input and check "Emulate three button mouse" option.
```
Comment (MattHeffron) [inline README.md:68]: Line 68: Unmatched right parenthesis: **difficulty.)**
Line 115: **at least in the library, lispusers.** (poor grammar) => **at least in the library and lispusers directories.**

**Review by MattHeffron — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MattHeffron) [reply README.md:68]: Apparently, I cannot add the comment to lines that were not modified. So, the line 115 change is here.

Comment (nbriggs) [reply README.md:68]: Also, "the scroll-wheel on some mice act**s** as one, with some difficulty.

I'll fix your comments and push a new version (BTW, Matt - I believe that you can update these branches too)

**Review by masinter — COMMENTED** [review]
thank you for fixing my mistakes... PR also needs approval to merge

File: README.md (hunk 1 above)
Comment (nbriggs) [reply README.md]: I think these are resolved now.  Marking it as such.

**Review by nbriggs — APPROVED** [review]
Looks good enough for now.

**Pull request merged**